src/schema/validator.ts

```diff
@@ -162,52 +154,22 @@ export class SchemaValidator {
 
   /**
    * Get a string representation of the schema validation errors.
+   * Adds additional (human friendly) information to certain errors.
    *
-   * @param errors An array of JsenValidateError objects.
-   * @param schema The validation schema.
+   * @param errors An array of AJV (DefinedError) objects.
    */
-  private getErrorsText(errors: JsenValidateError[], schema: JsonMap): string {
+  private getErrorsText(errors: DefinedError[]): string {
     return errors
       .map((error) => {
-        // eslint-disable-next-line no-useless-escape
-        const property = RegExp(/^([a-zA-Z0-9\.]+)\.([a-zA-Z0-9]+)$/).exec(error.path);
-
-        const getPropValue = (prop: string): Optional<AnyJson> => {
-          const reducer = (obj: Optional<AnyJson>, name: string): Optional<AnyJson> => {
-            if (!isJsonMap(obj)) return;
-            if (isJsonMap(obj.properties)) return obj.properties[name];
-            if (name === '0') return asJsonArray(obj.items);
-            return obj[name] || obj[prop];
-          };
-          return error.path.split('.').reduce(reducer, schema);
-        };
-
-        const getEnumValues = (): string => {
-          const enumSchema = asJsonMap(getPropValue('enum'));
-          return (enumSchema && getJsonArray(enumSchema, 'enum', []).join(', ')) || '';
-        };
+        const msg = `${error.schemaPath}: ${error.message}`;
```

Comment:
Sorry to jump in randomly, but, if you haven't already, you could look at using [better-ajv-errors](https://www.npmjs.com/package/better-ajv-errors) to get nicely formatted errors messages out of ajv.

Reply:
Hey @smithgp, I had not seen that package and it does look like it does its job well. Although, we are swapping out JSEN for AJV because of [a vulnerability](https://github.com/bugventure/jsen/issues/85), because of that we are trying to reduce any additional dependencies if we can. I think what we've got here is good enough for our use case. Thanks for sharing, always feel free to jump up! 😄